doc/data/messages/d/dangerous-default-value/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
With a mutable default value, with each call the default value is modified, i.e.: 

.. code-block:: python
    whats_on_the_telly()  # ["property of the zoo"]
    whats_on_the_telly()  # ["property of the zoo", "property of the zoo"]
    whats_on_the_telly()  # ["property of the zoo", "property of the zoo", "property of the zoo"]